arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/continuations/EffectSpec.kt

```diff
@@ -302,7 +302,7 @@ class EffectSpec :
     "catch - error path and recover" {
       checkAll(Arb.int().suspend(), Arb.string().suspend()) { int, fallback ->
         effect<Int, String> {
-          raise<String>(int())
+          raise(int())
           fail("It should never reach this point")
```

Comment:
This line now warns about unreachable code. Same in the other similar tests. Is this line still useful, or should it be removed?

Reply:
Yes, that test is still useful. It verifies that passed `raise` no code will be executed, which is also confirmed by the IDEA inspection now 🥳

So, I think we should suppress this warning if possible. (Not sure what `@Supress` is needed for this).